app/src/main/java/com/severianfw/picto/data/repository/PhotoRepositoryImpl.kt

```diff
@@ -16,11 +23,24 @@ class PhotoRepositoryImpl @Inject constructor(
     }
 
     override fun getPhotos(page: Int): Single<List<PhotoResponse>> {
-        return apiService.getPhotos(CLIENT_ID, PAGE_SIZE, page)
+        return apiService.getPhotos(CLIENT_ID, PAGE_SIZE, page).doOnSuccess {
+            CoroutineScope(Dispatchers.IO).launch {
+                val photos = PhotoMapper.mapToPhotoItemModel(it)
+                photoDao.insertPhotos(photos)
+            }
+        }
     }
 
     override fun searchPhotos(page: Int, photoName: String): Single<SearchPhotoResponse> {
         return apiService.searchPhotos(CLIENT_ID, PAGE_SIZE, page, photoName)
     }
 
+    override fun getLocalPhotos(): Single<List<PhotoItemModel>> {
+        return photoDao.getPhotos()
+    }
+
+    override fun clearLocalPhotos() {
```

Comment:
I think it better we name it with the same context as wel..
Which is `deleteLocalPhotos()`